packages/metro/src/shared/output/bundle.flow.js

```diff
@@ -49,7 +49,7 @@ async function saveBundleAndMap(
     ...
   },
   options: OutputOptions,
-  log: (...args: Array<string>) => void,
```

Comment:
@szymonrybczak Hmm, can we drop this change? Rest looks good!

The interface here is that we expect a logger that is similar to `console.log()`'s API, which accepts multiple args that will be concatenated without a `,`.

- Happy with the other changes, which make this behaviour the most predictable wherever Metro is integrated.
- I'd prefer an upstream alignment in `@react-native-community/cli-tools`to `logger.info`. Our current types in `dev-middleware` assume this: https://github.com/facebook/react-native/blob/c0b5e2e031f0e471b57640be13fe086d36c48a99/packages/dev-middleware/src/types/Logger.js#L12-L15.

Reply:
Yeah, I'm good with deleting this change. We just need to be careful when adding more logging to check if they're correctly formatted 👍

Reply:
Removed in https://github.com/facebook/metro/pull/1300/commits/6f86b48aa16ed8a2b38bfbb3408b6040a82fe001